gitgud/__main__.py

```diff
@@ -42,7 +42,7 @@ def __init__(self):
         instructions_parser = self.subparsers.add_parser('instructions', help='Show the instructions for the current level', description='Show the instructions for the current level')
         reset_parser = self.subparsers.add_parser('reset', help='Reset the current level', description='Reset the current level')
         test_parser = self.subparsers.add_parser('test', help='Test to see if you\'ve successfully completed the current level', description='Test to see if you\'ve successfully completed the current level')
-        progress_parser = self.subparsers.add_parser('progress', help='Continue to the next level', description='Continue to the next level')
+        progress_parser = self.subparsers.add_parser('progress', help='Continue to the next level', description='This command is now deprecated. To continue to the next level please use `git gud load` instead.')
```

Comment:
We're in v0.1 pre-release right now, so no need to worry about deprecation, go ahead and delete this line. Same goes for anything else related to the progress command